components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml

```diff
@@ -1533,6 +1532,28 @@ components:
         self:
```

Comment:
let's add this attribute as the last in the object definition

Reply:
fixed